html/assets/compare/machinery.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function () {
+  // Render the diff
+  var diff = getDiff();
+
+  templates = {};
+  scopes = [
+    "os",
+    "os_partial",
+    "changed_managed_files",
+    "changed_managed_files_partial",
+    "config_files",
+    "config_files_partial",
+    "unmanaged_files",
+    "groups",
+    "users",
+    "packages",
+    "packages_partial",
+    "patterns",
+    "repositories",
+    "repositories_partial",
+    "services"
+  ];
+
+  $.each(scopes, function(index, scope) {
+    templates[scope] = Hogan.compile($("#scope_" + scope).html());
+  });
+
+  other_templates = [
+    "description_a",
+    "description_b",
+    "both_descriptions"
+  ];
+  $.each(other_templates, function(index, template) {
```

Comment:
Identifier 'other_templates' is not in camel case.<br>'other_templates' is not defined.